### DISCUSSION TITLE
Solution to add multiple mods at once

### DISCUSSION THREAD

**Discussion by Buck-Irl** [gameserverapp/Platform#831] [Suggestions]
Like the title says, if we had a way to copy all the mod id numbers at once and install them that way,  rather having to search them one by one. 

Alternatively, a way to copy all active mods from one server to another would do the job. 
![Screenshot_20240423_123604_Chrome](https://github.com/gameserverapp/Platform/assets/162809925/74fbd01c-5bf6-43e8-8ff3-422e9297b446)

**Comment by LimitlessDrag0n** [comment]
You can use all mods from a different server and put it in there

![image](https://github.com/gameserverapp/Platform/assets/99832504/23d5b9cf-161e-4658-ae11-0de65cbb06e0)

Also if you have multiple mods and their ids, you can put them in there seperated by comma. If you mean you want something like a browser, I guess for steam it's harder to implement

**Reply by Buck-Irl** [reply]
Is that ASE?  There isn't option for that on ASA servers.

**Comment by mvaessen** [comment]
created dev item #960

**Comment by mvaessen** [comment]
Thanks for the suggestion, this will be available later today
